cmd/helm/release_testing.go

```diff
@@ -36,25 +37,43 @@ The tests to be run are defined in the chart that was installed.
 
 func newReleaseTestCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	client := action.NewReleaseTesting(cfg)
-	var outfmt output.Format
+	client.Namespace = settings.Namespace()
```

Comment:
Would you mind moving this to run inside `RunE`, right before we call `client.Run`? I fear there may be a risk of flag parsing race conditions occurring if we set the namespace when the commands are being assembled. We may want to set the namespace parameter when the command is actually executed like we do for other commands:

https://github.com/helm/helm/blob/c3e356faba0c23626a4eb84c3bb3656279468585/cmd/helm/upgrade.go#L74-L79

Reply:
Good point. Done!